### PR TITLE
Somehow this fixes detection of the availability of struct timeval fo…

### PR DESCRIPTION
**Pull request by RincewindsHat**
…r me

### CODE REVIEW

**Comment by RincewindsHat** [comment]
@sni opinions?

**Comment by sni** [comment]
what was the actual issue?

**Comment by RincewindsHat** [comment]
I got a compiler warning for have `struct timeval` redefined inside the project (`plugins/utils.h:55`) (it was already available on my system).
The part in the `configure.ac` checks for the availability of this type, but for some reason it did not set `HAVE_STRUCT_TIMEVAL` on my system.

Separating the the configure statements did the trick for me, althoug I cannot say why.

**Comment by RincewindsHat** [comment]
@sni yes/no?

File: configure.ac [hunk 1]
```diff
@@ -644,12 +644,16 @@ AC_TRY_COMPILE([#include <sys/time.h>],
...
                AC_DEFINE(HAVE_STRUCT_TIMEVAL,1,[Define if we have a timeval structure])
-               AC_TRY_COMPILE([#include <sys/time.h>],
-                              [struct timeval *tv;
-                               struct timezone *tz;
-                               gettimeofday(tv, tz);],
-                              AC_DEFINE(HAVE_GETTIMEOFDAY,1,[Define if gettimeofday is found]),
-                              AC_DEFINE(NEED_GETTIMEOFDAY,1,[Define if gettimeofday is needed])))
+			   FOUND_STRUCT_TIMEVAL="yes")
+
+if test x$FOUND_STRUCT_TIMEVAL = x"yes"; then
```
Comment (datamuc) [inline configure.ac:649]:
```suggestion
if test "$FOUND_STRUCT_TIMEVAL" = yes; then
```

Comment (waja) [reply configure.ac:649]: Ähm ... good catch, look wired.

Comment (waja) [reply configure.ac:649]: Anyway ... https://github.com/monitoring-plugins/monitoring-plugins/pull/1927/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810L630 looks equal

Comment (RincewindsHat) [reply configure.ac:649]: isn't that how one has to compare strings to be the comparison does the right thing if one of the strings is empty?

Comment (waja) [reply configure.ac:649]: I'd opt for

```suggestion
if test x"$FOUND_STRUCT_TIMEVAL" = x"yes"; then
```

or

```suggestion
if test "x$FOUND_STRUCT_TIMEVAL" = "xyes"; then
```

**Review by waja — APPROVED** [review]
okay ... dunno why this is needed, but looks good from syntax (any styling) ;)

**Pull request merged**